battle/attack/AttackSimple.java

```diff
@@ -156,6 +156,12 @@ public void excuse() {
 			int wid = dire == 1 ? W_E_WID : W_U_WID;
 			float addp = (dire == 1 ? W_E_INI : W_U_INI) + wid / 2f;
 			float p0 = model.getPos() + dire * addp;
+
+			if (proc.WAVE.lv < 0 && model.getDire() != 0) {

```

Comment:
In line 160, what's the reason of checking dire here? This means that if dire is 0, inverted wave will work weirdly

Reply:
It was part of my initial fixes when I ran into bugs with wave positioning. Now it is redundant and will be removed.